src/firefly/js/tables/TableUtil.js

```diff
@@ -120,7 +120,7 @@ export function makeIrsaCatalogRequest(title, project, catalog, params={}, optio
     title = title || catalog;
     options.use = options.use || 'catalog_overlay';
     const tbl_id = options.tbl_id || uniqueTblId();
-    const id = 'GatorQuery';
+    const id = params.processorName||'GatorQuery';
```

Comment:
I did know that, so that I introduced a new parameter in my code "processorName", and the 
" const id = params.processorName||'GatorQuery';" will always return 'GatorQuery' except in Finderchart because the processorName  is only set in Finderchart.   All the existing calls will have "params.processorName" as undefined.   Of course, I can add one method to just do that.  But since this line does not hurt anything else, I just modified the existing method.